<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>219</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QMfgTool_MultiPanel</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
   <item>
    <widget class="QGroupBox" name="unassigned_groupBox">
     <property name="title">
      <string>Unassigned</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="drive_lineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="drive_label">
        <property name="text">
         <string>Drive(s):</string>
        </property>
        <property name="buddy">
         <cstring>drive_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="drive_horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="status_lineEdit">
        <property name="text">
         <string>No Device Connected</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <spacer name="unassigned_verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>313</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QProgressBar" name="item_progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QProgressBar" name="total_progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="statusInformation_verticalLayout" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="statusInformation_groupBox">
       <property name="title">
        <string>Status Information</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0" columnminimumwidth="0,50">
        <item row="0" column="0">
         <widget class="QLabel" name="sucessfullOperationText_label">
          <property name="text">
           <string>Successful Operations:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="sucessfullOperationValue_label">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="failedOperationText_label">
          <property name="text">
           <string>Failed Operations:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="failedOperationValue_label">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="failureRateText_label">
          <property name="text">
           <string>Failure Rate:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="failureRateValue_label">
          <property name="text">
           <string>0 %</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="statusInformation_verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="statusInformation_horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="start_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Start</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="buttons_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>drive_lineEdit</tabstop>
  <tabstop>status_lineEdit</tabstop>
  <tabstop>start_pushButton</tabstop>
  <tabstop>exit_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exit_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
